pkgs/applications/networking/remote/vmware-horizon-client/default.nix

```diff
@@ -70,7 +133,7 @@ in stdenv.mkDerivation {
     ln -s "${vmwareFHSUserEnv}/bin/vmware-view" "$out/bin/"
   '';
 
-  meta = with lib; {
+  meta = with stdenv.lib; {
```

Comment:
```suggestion
  meta = with lib; {
```

Reply:
I removed `lib` from the package arguments just to remove clutter up there, and typed `lib` down by the meta attribute instead. Would is be better to NOT do this, and add `lib` back to the package arguments?

Reply:
We recently changed the contributing guide which deprecates stdenv.lib and it shouldn't be used anymore.

Reply:
Aha, gotcha. Done.